packages/apps/docs/src/docs/build/authentication/register-an-account.md

```diff
@@ -2,36 +2,44 @@
 title: Register a Kadena SpireKey account
```

Comment:
I'm not sure about adding this guide in the midst of the other guides. Reason being that this guide is targeted for wallet devs while the other guides target dapp devs. Dapp devs should not need to be instructed how to recover accounts, users should not read these docs on how to recover accounts (other than a FAQ type of thing)

Reply:
I removed the implementation info from the recover topic.